### PR TITLE
Add: Media by 2TallTyler

### PR DESCRIPTION
**Pull request by 2TallTyler**
Adds the content I created for the Steam store page:
- Ten screenshots at 1280px by 720px, the size recommended by Steam
  - All NewGRFs shown have a GPL license (version 2 or 3) which allow this type of promotional use
  - Screenshots show OpenGFX base graphics, not TTD
- Small capsule (231px by 87px)

I also created a video trailer, but it is 112 MB and GitHub has a file limit of 100 MB. Is there another place it can be backed-up?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Translations for the Steam Store Page
 
-This repository contains the translations for the translatable text blocks on [OpenTTD's Steam Store Page](https://store.steampowered.com/app/1536610/OpenTTD/).
+This repository contains the media assets and translations for the translatable text blocks on [OpenTTD's Steam Store Page](https://store.steampowered.com/app/1536610/OpenTTD/).
```
Comment (frosch123) [inline README.md:3]: You forgot to adjust the headline :)

**Comment by TrueBrain** [comment]
Before I forget: the capsule is PNG only. I generated all capsules via GIMP, so we should attach the source file with it. I believe this capsule was done as counter-suggestion to the one I have; possibly I duplicated it via GIMP. Either way, only adding the PNG is something we shouldn't do :)

But this is something for me to figure out!

**Comment by 2TallTyler** [comment]
I added my GIMP source file. It's unresized, so let me know if you want me to scale it down to the proper size instead.

**Comment by TrueBrain** [comment]
> 
> 
> I added my GIMP source file. It's unresized, so let me know if you want me to scale it down to the proper size instead.

No worries; it is very possible I took your PNG and imported it in my GIMP source :) As I said, I need to figure out the mess I made here (on my local disk) and make sure they are in sync with what I uploaded to Steam :P I will get to that :D

**Comment by 2TallTyler** [comment]
OK, no problem. Just let me know if there's anything else I can do.

**Pull request closed without merging**